loginLegacy/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
+    <application>
+        <activity
+            android:name=".LoginScreenActivity"
```

Comment:
You need intent filters for MAIN, etc.

Reply:
made it have a main intent filter and set it as a launcher intent, since its the login screen for legacy